src/OpenTelemetry.AutoInstrumentation.Logging/ILogger.cs

```diff
@@ -0,0 +1,107 @@
+// <copyright file="ILogger.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+
+namespace OpenTelemetry.AutoInstrumentation.Logging;
+
+#pragma warning disable CS1591
+[System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "Not")]
+public interface ILogger
```

Comment:
Shall we use a different name here?
`ILogger` is a dotnet reserved name and we could avoid naming conflict.

Reply:
I'll rename to IOtelLogger.